tests/test_test_utils.py

```diff
@@ -14,11 +15,20 @@
                                 teardown_test_loop, unittest_run_loop)
 
 
-def _create_example_app():
+_hello_world_str = "Hello, world"
```

Comment:
At this rate you can even generate a random string. Like:

```python
uuid.uuid4().hex
```

I would actually even put the str, bytes and gzip inside the app instance:

```python
app = web.Application()
app["body_str"] = uuid.uuid4().hex
app["body_bytes"] = app["body_str"].encode('utf-8')
app["body_gz"] = gzip.compress(app["body_bytes"])
```